### ISSUE TITLE
Can load and start new dynamic campaign. I fly first mission in DCS, then when I quit receive following script error.

### ISSUE THREAD

**Issue by Ballping** [dcs-liberation/dcs_liberation#29]
This is a brand new install of Liberation with no prior installs and DCS is installed on my C: drive at default directory. I've attempted to create a new campaign and have also attempted to delete and re-create save file. Also created a campaign using a different area of the Caucasus map.

![Script_Error_Liberation](https://user-images.githubusercontent.com/68836409/88501874-c083d580-cf9a-11ea-8b7d-e570b2200896.JPG)

**Comment by qwertymanisalive** [comment]
Two possible stumbling blocks are:

1) Make sure that you are pointing the program to the correct directories during install. The program will not work if you don't.

2.) Make sure you change the MissionScripting.lua file that is included in the install zip file

I made both these mistakes when I installed and got it working after I made sure I did these steps correctly.

**Comment by qwertymanisalive** [comment]
Two possible stumbling blocks are:

1. Make sure that you are pointing the program to the correct directories during install. The program will not work if you don't.
2. Make sure you change the MissionScripting.lua file that is included in the install zip file

I made both these mistakes when I installed and got it working after I made sure I did these steps correctly.

**Comment by Ballping** [comment]
> 
> 
> Two possible stumbling blocks are:
> 
>     1. Make sure that you are pointing the program to the correct directories during install. The program will not work if you don't.
> 
>     2. Make sure you change the MissionScripting.lua file that is included in the install zip file
> 
> 
> I made both these mistakes when I installed and got it working after I made sure I did these steps correctly.

Thanks for the reply. I actually did make the first mistake when I originally installed the program. I however corrected the game install pathing afterwards to the correct path.

I'm not sure what you mean by change the MissionScripting.lua file. I'll go back and re-read install instructions and see what I missed but if you could elaborate I'd be appreciative.

Thanks,

**Comment by Ballping** [comment]
>     2\. Make sure you change the MissionScripting.lua file that is included in the install zip file

Hello Qwertymaisalive, I just wanted to let you know that this was the clue that helped me resolve the issue. I found another video: https://www.youtube.com/watch?v=YIqDxKdo4as by Samurai 1-1 and this video + your comment allowed me to resolve the issue. 

Basically I had to comment out the MissionScript.lua file in my DCS install directory and that cleared the error and allowed DCS Liberation 2.0 to run successfully. 

I appreciate the assistance.
